### ISSUE TITLE
Safety throws error with positives

### ISSUE THREAD

**Issue by adamus1red** [sopel-irc/sopel#1850]
### Description
When Safety is enabled with a valid personal free API key whenever a new URL is seen by the bot in a safety enabled channel, it throws an error.

### Reproduction steps
1. Set `safety` to `on` or `strict`
2. Send new message containing a URL

### Expected behavior
What did you expect to happen instead?

### Logs
From the bot
```
<Ad> https://www.cnn.com/2020/04/21/uk/uk-death-toll-coronavirus-statistics-gbr-intl/index.html
<^quark^> [ UK coronavirus: Death toll in England and Wales 41% higher than government figures - CNN ] - www.cnn.com
<^quark^> Unexpected error ('positives') from Ad at 2020-04-21 21:47:29.752477. Message was: https://www.cnn.com/2020/04/21/uk/uk-death-toll-coronavirus-statistics-gbr-intl/index.html
```

From logs
```
[2020-04-21 21:47:29,752] sopel.bot            ERROR    - Unexpected error ('positives') from Ad at 2020-04-21 21:47:29.752477. Message was: https://www.cnn.com/2020/04/21/uk/uk-death-toll-coronavirus-statistics-gbr-intl/index.html
Traceback (most recent call last):
  File "/home/sopel/.local/lib/python3.8/site-packages/sopel-7.0.0-py3.8.egg/sopel/bot.py", line 590, in call
    exit_code = func(sopel, trigger)
  File "/home/sopel/.local/lib/python3.8/site-packages/sopel-7.0.0-py3.8.egg/sopel/modules/safety.py", line 166, in url_handler
    data = {'positives': result['positives'],
KeyError: 'positives'

```

### Environment
- Sopel `.version`: 7.0.1
- Sopel installed via: git 
- Python version: 3.8
- Operating system: alpine
- IRCd `/version`: UnrealIRCd-5.0.3.1
- Relevant plugins: safety

**Comment by dgw** [comment]
OK, this is a pretty basic one now that I've looked at the API documentation and examined some sample responses in Postman. VirusTotal only sends `"positives"` in the JSON response if it's already scanned the link, so that key (obviously) isn't always present.

We tell VT `scan=1`, so it will go off and scan the link we give it if there's no existing scan report. The `safety` plugin must check back later to get the results; there's no mechanism to be proactively notified when the scan we initiated completes.

Of note: A free/public API key may only send 4 queries per minute. An empty response with HTTP status code 204 is returned if any quota (minute, day, or month) has been exceeded for the provided key, but there's no information about when the over-limit condition will abate (Twitter sends this info, and it's really helpful).

VirusTotal's API v3.0 is in beta (we use v2.0). It provides a semantically correct 429 error when the quota has been exceeded, according to the documentation, but I wasn't able to trigger an over-limit condition. Maybe the quota restrictions are relaxed during the beta period; it didn't seem important enough to investigate further just yet.

----

**Anyway:** I think for 7.0.2, the solution is switching to `.get('key', <default>)` instead of subscripting the dictionary, so we gracefully handle missing keys. It's Good Enough™ to assume that we'll (almost) never be the first to submit a malicious URL.

For 7.1, we can get fancier. I think we _could_ do with a revamp of this plugin to submit URLs and then check for results in ~30 sec if it gets a "scan queued" response instead of an immediate result. Doing such a rewrite for 7.0.2 is too much, though; it's just a maintenance release.
